go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/motemen/gobump
+
+go 1.13
+
+require (
+	github.com/Masterminds/semver v1.5.0
```

Comment:
Why did you fix this to the older version? [StrictNewVersion is added in v3.0.0](https://github.com/Masterminds/semver/blob/910aa146bd66780c2815d652b92a7fc5331e533c/CHANGELOG.md#300-2019-09-12) and the command is now unavailable by go mod.
```
 % sh -c 'cd && env GO111MODULE=on go get github.com/motemen/gobump/cmd/gobump'
go: finding github.com/motemen/gobump latest
# github.com/motemen/gobump
.share/go/pkg/mod/github.com/motemen/gobump@v0.0.0-20191228120854-33441ed4ff7e/gobump.go:275:17: undefined: semver.StrictNewVersion
.share/go/pkg/mod/github.com/motemen/gobump@v0.0.0-20191228120854-33441ed4ff7e/gobump.go:279:16: undefined: semver.StrictNewVersion
.share/go/pkg/mod/github.com/motemen/gobump@v0.0.0-20191228120854-33441ed4ff7e/gobump.go:287:12: undefined: semver.StrictNewVersion
```